### DISCUSSION TITLE
搜索关键词中带"-"，显示搜索失败

### DISCUSSION THREAD

**Discussion by wuscloud** [tamlok/vnote#1964] [Q&A]
![image](https://user-images.githubusercontent.com/12063279/145139264-ac5d4356-f0ce-4783-b394-d1ec6af08c0b.png)

**Comment by feloxx** [comment]
请升级最新版本
![image](https://user-images.githubusercontent.com/8359848/148972224-fe9f1f5d-8950-45a4-9248-dc04de3f91df.png)

**Reply by tamlok** [reply]
-D会被解释为选项，用双引号
